backend/src/main/scala/cromwell/backend/io/GlobFunctions.scala

```diff
@@ -8,16 +8,15 @@ import common.validation.ErrorOr.ErrorOr
 import wom.values._
 import wom.expression.IoFunctionSet
 import wom.graph.TaskCallNode
-import wom.types.WomAnyType
 import wom.values.WomGlobFile
 
 trait GlobFunctions extends IoFunctionSet {
 
   def callContext: CallContext
 
   def findGlobOutputs(call: TaskCallNode, jobDescriptor: BackendJobDescriptor): ErrorOr[List[WomGlobFile]] =
-    call.callable.outputs.flatTraverse[ErrorOr, WomGlobFile] {
-      _.expression.evaluateFiles(jobDescriptor.localInputs, this, WomAnyType) map {
+    call.callable.outputs.flatTraverse[ErrorOr, WomGlobFile] { outputDefinition =>
+      outputDefinition.expression.evaluateFiles(jobDescriptor.localInputs, this, outputDefinition.womType) map {
```

Comment:
what's up with the type change?

Reply:
The FileEvaluator needs to know what it's coercing inputs into. Previously it was failing to read pair inputs from JSON (even though they didn't have any globs, they still need to go through the "does it have globs" method)